server/views/global/icons.config.js

```diff
@@ -0,0 +1,28 @@
+import path from 'path';
+import fs from 'fs';
+
+const iconsPath = path.join(__dirname, '../icons');
```

Comment:
We should remember that I/O creates performance problems.
What about outputting a JSON file?